### ISSUE TITLE
 Could not decrypt yara rule

### ISSUE THREAD

**Issue by davidep** [NextronSystems/thor-lite#1]
Hi, I'm having errors with the latest thor-lite. i'm using the license from the previous sparkcore... could that be an issue?

```
Error 1 | Mar 31 12:22:16 DESKTOP-rand/IP.IP.IP.IPTHOR: Error:MODULE: InitMESSAGE: Could not decrypt yara ruleSCANID: S-fTLsmJjWsmEFILE: D:\\thor10lite-win-pack\signatures\yara\thor-all.yas ERROR: could not decode session key: crypto/rsa: decryption error
-- | --
Error 2 | Mar 31 12:22:16 DESKTOP-rand/IP.IP.IP.IPTHOR: Error:MODULE: InitMESSAGE: Could not decrypt yara ruleSCANID: S-fTLsmJjWsmEFILE: D:\\thor10lite-win-pack\signatures\yara\thor-keywords.yas ERROR: could not decode session key: crypto/rsa: decryption error
Error 3 | Mar 31 12:22:16 DESKTOP-rand/IP.IP.IP.IPTHOR: Error:MODULE: InitMESSAGE: Could not decrypt yara ruleSCANID: S-fTLsmJjWsmEFILE: D:\\thor10lite-win-pack\signatures\yara\thor-log-sigs.yas ERROR: could not decode session key: crypto/rsa: decryption error
Error 4 | Mar 31 12:22:16 DESKTOP-rand/IP.IP.IP.IPTHOR: Error:MODULE: InitMESSAGE: Could not decrypt yara ruleSCANID: S-fTLsmJjWsmEFILE: D:\\thor10lite-win-pack\signatures\yara\thor-process-memory-sigs.yas ERROR: could not decode session key: crypto/rsa: decryption error
Error 5 | Mar 31 12:22:16 DESKTOP-rand/IP.IP.IP.IPTHOR: Error:MODULE: InitMESSAGE: Could not decrypt yara ruleSCANID: S-fTLsmJjWsmEFILE: D:\\thor10lite-win-pack\signatures\yara\thor-registry.yas ERROR: could not decode session key: crypto/rsa: decryption error
Error 6 | Mar 31 12:22:17 DESKTOP-rand/IP.IP.IP.IPTHOR: Error:MODULE: InitMESSAGE: Could not decrypt ioc fileSCANID: S-fTLsmJjWsmEFILE: D:\\thor10lite-win-pack\signatures\iocs\custom-evil-hashes.dat ERROR: could not decode session key: crypto/rsa: decryption error
Error 7 | Mar 31 12:22:17 DESKTOP-rand/IP.IP.IP.IPTHOR: Error:MODULE: InitMESSAGE: Could not walk through ioc signature directoriesSCANID: S-fTLsmJjWsmE ERROR: could not decode session key: crypto/rsa: decryption error
```

**Comment by Neo23x0** [comment]
It's likely a bug in the early v10.4.0 

We've fixed that in v10.4.1. You can upgrade your scanner with 

```
thor-lite-util.exe upgrade
```

**Comment by davidep** [comment]
yep, thanks! indeed it just worked after the upgrade.
Thanks again
